components/constants.ts

```diff
@@ -0,0 +1,55 @@
+import { getRelativePath } from '@/utils';
```

Comment:
We already has a `constants.ts` in the root project.....
Please use that file, and just add your enum.